netty/src/test/java/io/grpc/netty/NettyClientStreamTest.java

```diff
@@ -392,6 +393,38 @@ public void removeUserAgentFromApplicationHeaders() {
         .containsEntry(Utils.USER_AGENT, AsciiString.of("good agent"));
   }
 
+  @Test
+  public void getRequestSentThroughHeader() {
```

Comment:
Added a test in AbstractClientStream2Test.java. I kept this one the same though.